core/src/crypto/Wallet.ts

```diff
@@ -25,20 +27,33 @@ import { createRxDatabase, addRxPlugin } from 'rxdb/plugins/core'
 import moment from 'moment'
 import { EthereumAuthProvider } from '3id-connect'
 
+import * as PouchdbAdapterIdb from 'pouchdb-adapter-idb'
+import * as PouchdbAdapterMemory from 'pouchdb-adapter-memory'
+import { RxDBEncryptionPlugin } from 'rxdb/plugins/encryption'
+import { RxDBValidatePlugin } from 'rxdb/plugins/validate'
+import { RxDBUpdatePlugin } from 'rxdb/plugins/update'
+import { getWeb3 } from '../utils/web3factory'
+addRxPlugin(RxDBDevModePlugin)
```

Comment:
I think this should be removed: we do not want to execute "Dev Mode" plugin in production. Besides, sometimes it activates twice.